JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -219,17 +219,25 @@ private fun BulletParagraph(
     paragraphStyle: ParagraphStyle
 ) {
     Row {
-        Box(
-            modifier = LayoutSize(8.dp, 8.dp),
-            backgroundColor = contentColor(),
-            shape = CircleShape
-        ) {
-            // empty box
+        with(DensityAmbient.current) {
+            // this box is acting as a character, so it's sized with font scaling (sp)
+            Box(
+                    modifier = Modifier
+                            .preferredSize(8.sp.toDp(), 8.sp.toDp())
+                            .alignWithSiblings {
+                                // this makes the baseline of the Box be 9.sp from the top
+                                9.sp.toIntPx()
```

Comment:
Why 9sp??

Reply:
My comment there is terrible :). Rewriting it to answer this question!

Reply:
```
                            // Add an alingment "baseline" 1sp below the bottom of the circle
```